test/handler/test_phpstan_handler.vader

```diff
@@ -38,37 +40,5 @@ Execute(Output with some errors should be parsed correctly):
   \   }
   \ ],
   \ ale_linters#php#phpstan#Handle(bufnr(''), [
-  \   'phpstan-test-files/foo/test.php:9:Call to method format() on an unknown class DateTimeImutable.',
-  \   'phpstan-test-files/foo/test.php:16:Sample message.',
-  \   'phpstan-test-files/foo/test.php:192:Invalid command testCommand.',
-  \])
-
-Execute(Output should be parsed correctly with Windows paths):
-  call ale#test#SetFilename('phpstan-test-files/foo/test.php')
-
-  AssertEqual
-  \ [
-  \   {
-  \     'lnum': 9,
-  \     'text': 'Access to an undefined property Test::$var.',
-  \     'type': 'E'
-  \   }
-  \ ],
-  \ ale_linters#php#phpstan#Handle(bufnr(''), [
-  \   'D:\phpstan-test-files\foo\test.php:9:Access to an undefined property Test::$var.',
-  \])
-
-Execute(Output for .inc files should be parsed correctly):
-  call ale#test#SetFilename('phpstan-test-files/test.inc')
-
-  AssertEqual
-  \ [
-  \   {
-  \     'lnum': 9,
-  \     'text': 'Access to an undefined property Test::$var.',
-  \     'type': 'E'
-  \   }
-  \ ],
-  \ ale_linters#php#phpstan#Handle(bufnr(''), [
-  \   '/phpstan-test-files/foo/test.inc:9:Access to an undefined property Test::$var.',
+  \ '{"totals":{"errors":0,"file_errors":11},"files":{"'.expand('%:p').'":{"errors":3,"messages":[{"message":"Call to method format() on an unknown class DateTimeImutable.","line":9,"ignorable":true},{"message":"Sample message.","line":16,"ignorable":true},{"message":"Invalid command testCommand.","line":192,"ignorable":true}]}}}',
```

Comment:
Apparently this does not work as expected on Windows, not sure how to debug it.

Reply:
When I try troubleshooting this with `echom`, it hangs forever :disappointed:

Reply:
This is what the dictionary looks like on Windows:

`{"C:\testplugin\testphpstan-test-files\foo\test.php":{"messages":[{"line":9,"message":"Call to method format() on an unknown class DateTimeImutable.","ignorable":true},{"line":16,"message":"Sample message.","ignorable":true},{"line":192,"message":"Invalid command testCommand.","ignorable":true}],"errors":3}}`

Notice the missing backslash between `C:\testplugin\test` and `phpstan-test-files\foo\test.php`